### DISCUSSION TITLE
Adding apps to folders

### DISCUSSION THREAD

**Discussion by JacobHornbeck** [LawnchairLauncher/lawnchair#3370] [Q&A] [answered]
In an older version (I think I was using version 2 (with XIcons), but I am not sure), the only way to add apps to folders (in the app drawer) was through the settings. In version 12.1, I can't even find any way to organize apps in the app drawer. Is this possible? And am I just missing it? Or was this option removed from the app?

**Comment by ghost** [comment]
The feature wasn't removed from the app, it's just not implemented yet. They are building the new version from scratch so we will just have to wait till they're done adding stuff and polishing up the software.

**Comment by SuperDragonXD** [comment, marked as answer]
Hello! I know that this reply is a bit late, but we now support app drawer folders in Lawnchair 15+.
